packages/manager/apps/public-cloud/src/components/beta-warning/beta-warning.html

```diff
@@ -0,0 +1,39 @@
+<div class="p-4" data-ng-if="!$ctrl.isHidden">
+    <h1 class="d-flex logo">
+        <i class="oui-icon oui-icon-ovh" style="font-size: 5rem;"></i>
+        <span class="ml-3 my-auto">OVH</span>
+    </h1>
+    <div class="container-fluid">
+        <div class="row d-flex justify-content-center">
+            <div class="col-xs-10 col-md-6 col-lg-5">
+                <div class="text-center mb-4">
+                    <img class="mb-4" data-ng-src="{{ $ctrl.illustration }}" alt="" style="width: 8vw;" />
```

Comment:
Empty `alt`? 
Also I think you can avoid inline style with `boostrap` utilities https://getbootstrap.com/docs/4.3/utilities/sizing/#relative-to-the-viewport